packages/editor/src/components/post-status/index.js

```diff
@@ -200,13 +207,15 @@ export default function PostStatus() {
 							variant="tertiary"
 							size="compact"
 							onClick={ onToggle }
+							icon={ postStatusesInfo[ status ]?.icon }
+							iconSize="20"
```

Comment:
Without the `iconSize`, the row get a bit bigger than the 32px, which is like a `min-height` for rows. Any better alternatives are welcome.

Reply:
It's because `.editor-post-panel__row-control .components-button` was overriding the `height` value. Removing that fixed it, but revealed that the 'Parent' button was not the correct size, so I fixed that too.